### ISSUE TITLE
Automatic search when editing filters

### ISSUE THREAD

**Issue by OzoneGrif** [pacien/ldgallery#146]
it would be better to update the item grid as filters are being added and removed instead of requiring the user to hit the search button manually. It is quite nice to have a "live" view of the search result, consistently with the already automatically updated suggestions. On the other hand, displaying the filtered list of items only once pressing the "search" button is a requirement for the "highlight" feature. However I remember that we thought about dropping that other mode altogether in the past. I propose that we update the search result automatically as filters are being edited until we get the "highlight" mode complete and ready in a later version.

**Comment by pacien** [comment]
Late thoughts: we could have some configuration option in config.json to either have the "search by highlight automatically and hit the search button to filter" or "filter automatically, disabling search by highlight" behaviours.

**Comment by OzoneGrif** [comment]
Issue #168 should suffice to fix this issue.
I don't really want to go back to the previous "Navigation / Search" behavior, which was not very good either.

**Comment by pacien** [comment]
Pressing enter twice as allowed by #168 is indeed straightforward enough and eliminates the need for an additional setting. Let's close this.
